apps/block_scout_web/lib/block_scout_web/views/api/v2/fee_payment_view.ex

```diff
@@ -0,0 +1,23 @@
+defmodule BlockScoutWeb.API.V2.FeePaymentView do
+  use BlockScoutWeb, :view
+
+  def render("message.json", assigns) do
+    ApiView.render("message.json", assigns)
+  end
+
+  def render("fee_payments.json", %{fee_payments: fee_payments, next_page_params: next_page_params}) do
+    %{"items" => Enum.map(fee_payments, &prepare_fee_payments(&1, nil)), "next_page_params" => next_page_params}
+  end
+
+  def prepare_fee_payments(fee_payments, _conn) do
```

Comment:
`def prepare_fee_payment(fee_payment, _conn) do` is more clear bc it is formatting a single fee payment

Reply:
agree, method name updated